test/Microsoft.TestPlatform.AcceptanceTests/AcceptanceTestBase.cs

```diff
@@ -45,12 +45,20 @@ public class AcceptanceTestBase : IntegrationTestBase
     public const string DesktopRunnerTargetRuntime = "win7-x64";
     public const string CoreRunnerTargetRuntime = "";
     public const string InIsolation = "/InIsolation";
-
+    
     public const string NETFX452_48 = "net452;net461;net472;net48";
     public const string NETFX451_48 = "net452;net461;net472;net48";
     public const string NETCORE21_50 = "netcoreapp2.1;netcoreapp3.1;net5.0";
     public const string NETFX452_NET50 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1;net5.0";
     public const string NETFX452_NET31 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1";
+    public const string DEFAULT_RUNNER_NETFX = "net451";
+    /// <summary>
+    /// Our current defaults for .NET and .NET Framework.
+    /// </summary>
+    public const string DEFAULT_RUNNER_NETFX_AND_NET = $"{DEFAULT_RUNNER_NETFX};netcoreapp2.1"; 
+    public const string DEFAULT_HOST_NETFX_AND_NET = "net451;netcoreapp2.1";
+    public const string LATEST_LEGACY = "Latest;LatestStable;LatestPreview;MostDownloaded;PreviousStable;LegacyStable";
```

Comment:
Maybe 

```suggestion
    public const string PREVIEW_TO_LEGACY_VERSIONS = "LatestPreview;LatestStable;MostDownloaded;PreviousStable;LegacyStable";
```

(not sure what the `Lastest` entry was supposed to correspond to).

Reply:
The previous code was correct (except for the ordering of preview after stable). Latest is an unreleased (most likely locally built) version e.g. 17.2.0-dev.